Commands/Commands/utils.js

```diff
@@ -570,6 +586,14 @@ function wait(bot, msg) {
   })
 }
 
+function delay(delayMS) {
+  return function(arg){
```

Comment:
Why return 2 function statements if you don't do anything with it? Just return the promise.